ci/jobs/mirror.yml

```diff
@@ -0,0 +1,44 @@
+resources:
+  - name: dwpdigital-example
+    type: registry-image-resource
+    source:
+      repository: dwpdigital/example
+    check_every: 1h
```

Comment:
might want this reducing, I think it was set to an hour when re needed to reduce the rate from the older Concourse platform